neqo-client/src/main.rs

```diff
@@ -528,29 +631,46 @@ impl<'a> Handler<'a> {
         }
         true
     }
+}
+
+fn to_headers(values: &[impl AsRef<str>]) -> Vec<Header> {
+    values
+        .iter()
+        .scan(None, |state, value| {
```

Comment:
Would slice::chunks() be easier to use here?

That is, `values.chunks(2).map(|c| Header::new(c[0].as_ref(), c[1].as_ref()))`  (That would panic if the number of values is not divisible by two, but that should not be allowed by the argument parser.